exercises/practice/high-scores/.docs/instructions.append.md

```diff
@@ -0,0 +1,7 @@
+# ignore
```

Comment:
What does this do?

Reply:
It appends the text to the existing instructions. This allows the instructions.md to be synced from problem-spec without overwriting track-specific instructions. 
* https://exercism.org/docs/building/tracks/practice-exercises#h-documentation-files

It has an h1 header to conform with [exercism's markdown rules](https://exercism.org/docs/building/markdown/markdown#h-headings)

A live example:
* https://exercism.org/tracks/jq/exercises/raindrops
* https://github.com/exercism/jq/tree/main/exercises/practice/raindrops/.docs